### PR TITLE
including MavenCode

### PR DESCRIPTION
**Pull request by charlesa101**
Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [ ] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

**Review by alexmt — CHANGES_REQUESTED** [review]
Thank you @charlesa101 ,

Can you please revert numbering changes (use `1.` on each line)

**Review by jessesuen — CHANGES_REQUESTED** [review]
Can you change the numbering to use `1.` across the board? Otherwise, it becomes unmaintainable?

EDIT: oops, I see alex had same comment

File: USERS.md [hunk 1]
```diff
@@ -90,111 +90,112 @@ Currently, the following organizations are **officially** using Argo CD:
...
+73. [VSHN - The DevOps Company](https://vshn.ch/)
+74. [Walkbase](https://www.walkbase.com/)
+75. [Wehkamp](https://www.wehkamp.nl/)
+76. [WeMo Scooter](https://www.wemoscooter.com/)
+77. [Webstores](https://www.webstores.nl)
+78. [Whitehat Berlin](https://whitehat.berlin) by Guido Maria Serra +Fenaroli
+79. [Witick](https://witick.io/)
+80. [WooliesX](https://wooliesx.com.au/)
+81. [Woolworths Group](https://www.woolworthsgroup.com.au/)
+82. [WSpot](https://www.wspot.com.br/)
```
Comment (pasha-codefresh) [inline USERS.md:173]: please revert numeration changes

**Review by pasha-codefresh — CHANGES_REQUESTED** [review]

**Comment by crenshaw-dev** [comment]
Closing because stale.

**Pull request closed without merging**